### ISSUE TITLE
LGPLv2.0+ license is not detected

### ISSUE THREAD

**Issue by rakeshbalusa** [nexB/scancode-toolkit#197]
OS : windows 7 - 64 bit
I've updated my scancode to the latest version available.

RPM file download URL - http://vault.centos.org/6.3/os/x86_64/Packages/libwmf-devel-0.2.8.4-22.el6.centos.x86_64.rpm

issue file path : `libwmf-devel-0.2.8.4-22.el6.centos.x86_64.rpm-extract\usr\include\libwmf\api.h`

The `license notice from line 4 to 17 is ignored` and it belongs to LGPL 2.0 or later. 
```
  The libwmf Library is free software; you can redistribute it and/or
   modify it under the terms of the GNU Library General Public License as
   published by the Free Software Foundation; either version 2 of the
   License, or (at your option) any later version.

   The libwmf Library is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
   Library General Public License for more details.

```

**Comment by pombredanne** [comment]
@rakeshbalusa you wrote in #190 :
>  I think we need to change this rule to a template. In that way we can solve all issues like #197 . Do I have to create a new ticket and pull request or is there any other way you want me to use?

Please go ahead and create a template all right eventually consider updating the rule you contributed  in #190

**Comment by balusarakesh** [comment]
fixed in #248 .
